receiver/prometheusreceiver/DESIGN.md

```diff
@@ -160,40 +161,39 @@ type Appender interface {
 One can see that the interface is very simple, it only has 4 methods: `Add`,
 `AddFast`, `Commit` and `Rollback`. The last two methods are easy to
 understand: `Commit` is called when the processing of the scraped page is
-completed and success, whereas `Rollback` is called if error occurs in between
+completed and successful, whereas `Rollback` is called if an error occurs during
 the process.
 
-However for the two methods starting with 'Add', there's no document on the
+However, for the two methods starting with 'Add', there's no document on the
```

Comment:
We will need to rewrite this section because AddFast is now removed. Let's follow up with another PR.